new-frontiers-14/frontier-station-14
# Changed the detective's spawn clothing options

## About the PR


Added a grey fedora, detective noir jumpsuit and jumpskirt to the detective's loadout.

## Why / Balance

Everything I added was already available in the DetDrobe, I just made it so that a detective does not need to go to their office for a grey fedora or noir jumpsuit.

## How to test

1. Open the detective loadout page.
2. Note the new items in the head and jumpsuit categories.
3. Choose any of them.
4. Spawn as detective and note that you spawn with what you chose.

## Requirements




**Changelog**


:cl:
- add: Added new clothes to the detective's loadout.
